rpc/v8/estimate_fee.go

```diff
@@ -259,35 +258,35 @@ curl --location 'http://localhost:6060/rpc/v0_8' \
 }
 */
 
-func (h *Handler) EstimateFee(broadcastedTxns []BroadcastedTransaction,
-	simulationFlags []rpcv6.SimulationFlag, id BlockID,
+func (h *Handler) EstimateFee(
+	broadcastedTxns []BroadcastedTransaction, simulationFlags []rpcv6.SimulationFlag, id *BlockID,
```

Comment:
nit: I think this is something we currently don't have convention, but I personally think this would help readability:
```suggestion
	broadcastedTxns []BroadcastedTransaction,
	simulationFlags []rpcv6.SimulationFlag,
	id *BlockID,
```
Reasons:
- Imagine functions where we have more arguments, it's easier to find the Nth argument. 
- It's more natural to see the latter argument on the right hand side of former argument in 1 line, or below the former argument in multiple lines. If we have 6 arguments in 2 lines, each has 3, the 4th argument is to the left of the 3rd argument.